File: javascript/examples/get_resources.js
const baseRequest = require('../base_request.js').baseRequest
const path = require('path');
process.chdir(path.resolve(__dirname, '../../'));
require('dotenv').config();

// Example of GET /resources

token = baseRequest.request.token({ path: '/resources' });
baseRequest.request.get({ path: '/resources', token: token });

File: python/examples/get_resources.py
import pprint
import os
import sys

sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))

from request import BaseRequest

# Example of GET /resources

class GetResourcesRequest(BaseRequest):
    def __init__(self):
        super().__init__()
    
    def get_resources(self):
        return self.make_request('GET', '/resources')

# run as standalone script by passing any command line argument
if len(sys.argv) > 1:
    request = GetResourcesRequest()
    response = request.get_resources()

    pprint.pprint(response)